.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "note-c"]
 	path = note-c
-	url = ../note-c.git
+	url = https://github.com/blues/note-c.git
```

Comment:
What advantage is brought forth from an absolute path of the submodule. IIRC, there is some inflexibility caused by moving away from the relative path.